Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -906,6 +909,10 @@ protected theorem AnalyticOn.continuousOn {s : Set E} (hf : AnalyticOn 𝕜 f s)
   fun x hx => (hf x hx).continuousAt.continuousWithinAt
 #align analytic_on.continuous_on AnalyticOn.continuousOn
 
+/-- Analytic everywhere means continuous -/
```

Comment:
For the docstring, I'd say "imply continuous" rather than "means".

Reply:
Done.